app/src/main/res/values/strings.xml

```diff
@@ -303,5 +303,6 @@
 
     <string name="error_no_custom_emojis">Your instance %s does not have any custom emojis</string>
     <string name="copy_to_clipboard_success">Copied to clipboard</string>
+    <string name="performing_lookup_title">Performing lookup...</string>
```

Comment:
This sounds a little tech-ish too me, maybe something more friendly like "looking it up?" Not sure what is easier to translate too.
Somebody ask people please, I'm not good at this
Also there's a symbol for ellipsys: … :3

Reply:
I have no strong feelings; it can be changed to whatever seems best.
(We're not using the ellipsis character in "Sending Toot..." :wink: )